<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>460</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft JhengHei UI</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>AutoDrawingBot</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>450</width>
      <height>351</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="controlpannel">
     <attribute name="title">
      <string>控制台</string>
     </attribute>
     <widget class="QPushButton" name="clipboard_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>剪貼簿</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="START_COORDy">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>60</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximum">
       <number>4800</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>終點位置 x, y</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>起始位置 x, y</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="START_COORDx">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>60</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximum">
       <number>4800</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="END_COORDx">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>60</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximum">
       <number>4800</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="END_COORDy">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>40</y>
        <width>60</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximum">
       <number>4800</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>簡化量</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="SCALINE_FACTOR">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximum">
       <number>500</number>
      </property>
      <property name="value">
       <number>23</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>縮放比例</string>
      </property>
     </widget>
     <widget class="QPushButton" name="resize_btn">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>240</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>縮放圖片</string>
      </property>
     </widget>
     <widget class="QPushButton" name="openfile_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>開啟檔案</string>
      </property>
     </widget>
     <widget class="QPushButton" name="setcoord_btn">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>210</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>調整座標</string>
      </property>
     </widget>
     <widget class="QPushButton" name="start_btn">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>210</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>開始</string>
      </property>
     </widget>
     <widget class="QListWidget" name="edges_select">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>40</y>
        <width>110</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>游標延遲</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="PGPAUSE">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="decimals">
       <number>5</number>
      </property>
      <property name="maximum">
       <double>1000.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.001000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="RESIZE_SCALE">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>100</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="prefix">
       <string/>
      </property>
      <property name="suffix">
       <string> %</string>
      </property>
      <property name="decimals">
       <number>2</number>
      </property>
      <property name="maximum">
       <double>500.000000000000000</double>
      </property>
      <property name="value">
       <double>100.000000000000000</double>
      </property>
     </widget>
     <widget class="QPushButton" name="resizeEdges_btn">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>240</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>縮放輪廓</string>
      </property>
     </widget>
     <widget class="QPushButton" name="reload_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>重新載入</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>TimeStamp</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Constants">
     <attribute name="title">
      <string>參數設置</string>
     </attribute>
     <widget class="QSpinBox" name="DEFAULT_SCALINE_FACTOR">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximum">
       <number>500</number>
      </property>
      <property name="value">
       <number>23</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="MIN_EDGE_LENGTH">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>7</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="KERNEL_SIZEx">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>60</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>17</number>
      </property>
      <property name="singleStep">
       <number>2</number>
      </property>
      <property name="value">
       <number>7</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="KERNEL_SIZEy">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>70</y>
        <width>60</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>17</number>
      </property>
      <property name="singleStep">
       <number>2</number>
      </property>
      <property name="value">
       <number>7</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="LOW_cTHRESHOLD">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>100</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>20</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="HIGH_cTHRESHOLD">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="DEFAULT_START_COORDy">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>160</y>
        <width>60</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximum">
       <number>4800</number>
      </property>
      <property name="value">
       <number>300</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="DEFAULT_START_COORDx">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>160</y>
        <width>60</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximum">
       <number>4800</number>
      </property>
      <property name="value">
       <number>70</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="DEFAULT_END_COORDy">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>190</y>
        <width>60</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximum">
       <number>4800</number>
      </property>
      <property name="value">
       <number>870</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="DEFAULT_END_COORDx">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>190</y>
        <width>60</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximum">
       <number>4800</number>
      </property>
      <property name="value">
       <number>1700</number>
      </property>
     </widget>
     <widget class="QCheckBox" name="AUTO_FIT_AREA">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>自動填滿區域</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>預設簡化量</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>最小邊緣長度</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>高斯核大小</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>邊緣檢測閾值(低)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>邊緣檢測閾值(高)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>預設起始位置</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>120</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>預設終點位置</string>
      </property>
     </widget>
     <widget class="QPushButton" name="apply_btn">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>220</y>
        <width>50</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>儲存</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="USE_SPEEDMODE">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>40</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>不顯示進度</string>
      </property>
      <property name="text">
       <string>啟用快速載入</string>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="image_display">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>350</width>
      <height>400</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>380</y>
      <width>300</width>
      <height>25</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="progressing_data">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>830</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
